code generators/de.fhdo.lemma.model_processing.code_generation.container_base/src/main/java/de/fhdo/lemma/model_processing/code_generation/container_base/buildscript/OpendRootBuildScript.xtend

```diff
@@ -0,0 +1,96 @@
+package de.fhdo.lemma.model_processing.code_generation.container_base.buildscript
+
+import de.fhdo.lemma.model_processing.code_generation.container_base.template.BuildScriptTemplate
+import java.util.Map
+import java.util.HashMap
+import java.nio.file.Files
+import java.nio.file.Paths
+import java.io.File
+import java.nio.charset.StandardCharsets
+import java.util.regex.Pattern
+
+/**
+ * This class contains the logic for the generation of the build scripts, used in the
+ * [BuildScriptCodeGenerator].
+ *
+ * @author [Philip Wizenty](mailto:philip.wizenty@fh-dortmund.de)
+ */
+class OpendRootBuildScript {
+    static OpendRootBuildScript instance
+    Map<String, String> buildScriptNodes
+
+    new () {
+        this.buildScriptNodes = new HashMap
+    }
+
+    /*
+     * Get or create instance of the [OpendRootBuildScript] class.
+     */
+    static def OpendRootBuildScript getInstance() {
+        if (OpendRootBuildScript.instance === null)
```

Comment:
Why precede `instance` with the class name`? This should not be necessary and increases reading effort.